packages/create-sitecore-jss/src/templates/react-native/.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+  "recommendations": [
+    "GraphQL.vscode-graphql",
+    "ms-azuretools.vscode-docker",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "rvest.vs-code-prettier-eslint",
+    "eamodio.gitlens",
+    "ms-vscode.PowerShell",
+    "mikestead.dotenv",
+    "davidanson.vscode-markdownlint",
+    "wix.vscode-import-cost",
+    "redhat.vscode-yaml",

```

Comment:
Could probably remove some of these for the React Native starter. At least the yml extension since there is no disconnected support.

Reply:
It supports disconnected mode, these files are defined using _json_ format instead of _yml_, but can be converted to yml,
but yes - in this case for now it's not needed to be added